RoddyCore/src/de/dkfz/roddy/client/RoddyStartupModeScopes.java

```diff
@@ -19,16 +19,30 @@
      * This is useful, if things can be handled in an easier way with scripts,
      * like i.e. file operations
      */
-    public static final int SCOPE_CLI = 0x00;
+    SCOPE_CLI,
 
     /**
      * This scope is for startup modes where only a reduced init is necessary.
      */
-    public static final int SCOPE_REDUCED = 0x10;
+    SCOPE_REDUCED,
 
     /**
-     * This scope is for modes with a full init.
+     * This scope is for modes with a full init. The job manager however will be the direct job manager.
      */
-    public static final int SCOPE_FULL = 0x20;
+    SCOPE_FULL(true),
 
+    /**
+     * This scope starts all modes and also initializes the Job Manager
+     */
+    SCOPE_FULL_WITHJOBMANAGER(true);
+
+    public final boolean needsFullInit;
```

Comment:
I see a conceptual problem here. 'full' kind of implies that the JM is also initialized as requested. As we have it in the moment, different configuration toggles/alternatives blend in this multifunctional enum. So the current design fails to implement the single responsibility principle.

We could initialize each start-up mode with just a set of toggles that indicate, whether or not specific features should be turned on or of (or independent enums, if its not just toggles). Would that make sense?

Reply:
As discussed, I'll leave it.